cloudfour/cloudfour.com-patterns
# Removes 👴 👵 retired icon assets

## Overview

This PR represents a small portion of the tasks described in #657. 

> ### Retire some icons
>
> There are a few icons we are never using or will likely stop using that we can remove:
> 
> - [x] dot: Was used by the radio pattern, which no longer exists since we never used it.
> - [x] lanyrd: This site has been broken since 2019.
> - [x] notification: It's unclear what this icon was meant to do that wasn't covered by notifications. It isn't used in any templates.

## Screenshots

n/a

## Testing

1. Confirm this PR removes; `dot.svg`, `lanyrd.svg` and `notification.svg`
2. View [theme deploy](https://deploy-preview-678--cloudfour-patterns.netlify.app) and ensure the above mentioned icons are not rendered in the list of available icon assets.
